eq-author/src/App/page/PropertiesPanel/GroupedAnswerProperties/AnswerProperties/Properties/TextProperties.test.js

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import { render, fireEvent, flushPromises } from "tests/utils/rtl";
+import actSilenceWarning from "tests/utils/actSilenceWarning";
+import TextProperties from "./TextProperties";
+import UPDATE_ANSWER_QUERY from "./updateAnswerMutation.graphql";
+import { colors } from "constants/theme";
+
+const renderTextProperties = (props, mocks) => {
+  return render(<TextProperties {...props} />, { mocks });
+};
+
+describe("Text Property", () => {
+  let queryWasCalled, props, mocks;
+
+  actSilenceWarning();
+
+  beforeEach(() => {
+    queryWasCalled = false;
+    props = {
+      answerId: "2",
+      maxLength: 2000,
+      hasMaxLengthOutRange: false,
+    };
+    mocks = [
+      {
+        request: {
+          query: UPDATE_ANSWER_QUERY,
+          variables: {
+            input: {
+              id: "2",
+              properties: {
+                maxLength: 2000,
+              },
+            },
+          },
+        },
+        result: () => {
+          queryWasCalled = true;
+          return {
+            data: {
+              updateAnswer: {
+                id: "2",
+                displayName: "aaaa",
+                label: "Vale 46",
+                type: "TextField",
+                properties: {
+                  required: false,
+                  maxLength: 2000,
+                },
+                validationErrorInfo: {
+                  id: "c0c7ebe7-4345-41cc-b248-b5d72c25fd9b",
+                  errors: [],
+                  totalCount: 0,
+                  __typename: "ValidationErrorInfo",
+                },
+                __typename: "BasicAnswer",
+              },
+            },
+          };
+        },
+      },
+    ];
+  });
+
+  it("on first render default value is 2000", () => {
```

Comment:
I think the pattern we follow for writing test text is something like 
```suggestion
  it("should render default value as 2000", () => {
```